simphony/cuds/model.py

```diff
@@ -0,0 +1,108 @@
+"""CUDS computational model implementation.
+
+This module contains the classes used to represent a computational model,
+based on SimPhoNy metadata.
+"""
+from ..core import DataContainer
+from .store import MemoryStateDataStore
+
+
+# IDEA: add consistency check handlers/instances to the CUDS class (dynamic).
+class CUDS(object):
+    """CUDS computational model."""
+    def __init__(self):
+
+        # Add datasets to memory datastore by default
+        self._dataset_store = MemoryStateDataStore()
+
+        # Memory storage to keep CUDS components
+        self._store = {}
+
+        # Data container
+        self._data = DataContainer()
```

Comment:
I have to rethink this part. The initial reason behind it is, well, everybody likes a `data` attribute! Apart from that, it is there to comply with CUDS metadata model. However, I see some inconsistencies here. Maybe the data attribute should reflect the items inside the internal store, but I have to see if it is feasible at all.